src/api/spec/cassettes/Packages/editing_package_files/editing_an_existing_file.yml

```diff
@@ -148,22 +150,94 @@ http_interactions:
     body:
       encoding: UTF-8
       string: |
-        <revision rev="1" vrev="1">
-          <srcmd5>fb5f41f8cd1fd100540acd508a41d045</srcmd5>
-          <version>unknown</version>
-          <time>1532183775</time>
-          <user>unknown</user>
-          <comment></comment>
-          <requestid/>
-        </revision>
+        <package name="branch_test_package" project="home:other_package_test_user">
+          <title>The Curious Incident of the Dog in the Night-Time</title>
+          <description>Autem qui dolores quam.</description>
+        </package>
     http_version: 
   recorded_at: Sat, 21 Jul 2018 14:36:15 GMT
+- request:
+    method: get
+    uri: http://backend:5352/source/home:package_test_user/file_edit_test_package?expand=1&rev=2
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+      Accept:
+      - "*/*"
+      User-Agent:
+      - Ruby
+  response:
+    status:
+      code: 200
+      message: OK
+    headers:
+      Content-Type:
+      - text/xml
+      Cache-Control:
+      - no-cache
+      Connection:
+      - close
+      Content-Length:
+      - '309'
+    body:
+      encoding: UTF-8
+      string: |
+        <directory name="file_edit_test_package" rev="2" vrev="2" srcmd5="e24708ab2d9432e02d7769bb911236ab">
+          <entry name="_config" md5="4208c36e853109ccf1593a3ac7d5533a" size="68" mtime="1532183787" />
+          <entry name="somefile.txt" md5="80e61b1d734da39f30872d0f55253418" size="58" mtime="1532183787" />
+        </directory>
+    http_version: 
+  recorded_at: Sat, 21 Jul 2018 14:36:27 GMT
+- request:
+    method: get
+    uri: http://backend:5352/source/home:package_test_user/file_edit_test_package/_service
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      Content-Type:
+      - text/plain
+      Accept-Encoding:
+      - identity
+      Accept:
+      - "*/*"
+      User-Agent:
+      - Ruby
+  response:
+    status:
+      code: 404
+      message: _service  no such file
```

Comment:
I don't know how to fix this... Not the haziest clue. Any help would be very appreciated.

Reply:
Upvote #5308 to ease test development would be my first advise

Reply:
if it was a 404 before then it should be ok...

Reply:
> Upvote #5308 to ease test development would be my first advise

Hm, that would make sense, yeah...

Reply:
> if it was a 404 before then it should be ok...

Its true, didn't notice...